vividus-bdd-engine/src/main/java/org/vividus/bdd/context/BddVariableContext.java

```diff
@@ -50,7 +50,11 @@ public void putVariable(Set<VariableScope> variableScopes, String variableKey, O
     @Override
     public void putVariable(VariableScope variableScope, String variableKey, Object variableValue)
     {
-        LOGGER.info("Saving a value '{}' into the '{}' variable '{}'", variableValue, variableScope, variableKey);
+        LOGGER.atInfo()
+                .addArgument(variableValue)
+                .addArgument(() -> variableScope.name().toLowerCase().replace('_', ' '))
```

Comment:
`variableScope.name().toLowerCase().replace('_', ' ')` why not do it in the enum ?

Reply:
already considered but:
-  override `toString` - may have unknown impact
- new method e.g. `toReadableString` -  this code snippet is used once and I don't see any other places where it could be useful

Reply:
but anyway you gave me an idea